test/CMakeLists.txt

```diff
@@ -514,6 +514,33 @@ COMMAND	$<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_ARGS} --verbose   --input 64
 COMMAND	$<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_ARGS} --verbose   --input 64	 32	7 7   --weights   192  32   3	3   --pads_strides_dilations	2   2	2   2	1   1
 )
 
+set(DYNAMIC_IMPLICITGEMM_ENVS
+    MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvAsmImplicitGemmV4R1DynamicFwd
+    MIOPEN_DEBUG_CONV_FFT=0
```

Comment:
When `MIOPEN_DEBUG_FIND_ONLY_SOLVER` is used, only `MIOPEN_DEBUG_CONV_FFT` and `MIOPEN_DEBUG_CONV_GEMM` must be set to 0. The reason is that FFT and GEMM are not reworked to Solvers yet (and thus do not obey to `MIOPEN_DEBUG_FIND_ONLY_SOLVER`).

Please remove `MIOPEN_DEBUG_CONV_DIRECT` and `MIOPEN_DEBUG_CONV_WINOGRAD`.

Reply:
OK, let me change it

Reply:
@atamazov I changed the code, please review again